coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/attachment/ProposalDevelopmentNarrativeRule.java

```diff
@@ -84,7 +84,8 @@ public boolean processAddNarrativeBusinessRules(AddNarrativeEvent narrativeEvent
         rulePassed &= getDictionaryValidationService().isBusinessObjectValid(narrative);
         rulePassed &= checkNarrative(document.getDevelopmentProposal().getNarratives(), narrative);
         rulePassed &= validFileNameCharacters(narrative);
-        
+        rulePassed &= validFileType(narrative);
+
         return rulePassed;
     }
```

Comment:
"Add" got omitted somehow. Should be there now, a little farther down in this file.